infra/prompt-flow/cwyd/output_parser.py

```diff
@@ -0,0 +1,47 @@
+from promptflow import tool
+import re
+
+
+def _clean_up_answer(answer: str):
+    return answer.replace("  ", " ")
```

Comment:
just out of curiosity - are we getting answers with 2 spaces ?

Reply:
Aha no idea! This is largely taken from the output_parser_tool https://github.com/Azure-Samples/chat-with-your-data-solution-accelerator/blob/main/code/backend/batch/utilities/parser/output_parser_tool.py

Reply:
ahh...